### PR TITLE
COMMERCE-8376 UI to show Shipment External Reference

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-8376

@luca-pellizzon

### CODE REVIEW

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrders.path [hunk 1]
```diff
@@ -95,6 +95,16 @@
...
+<tr>
+	<td>ORDER_AND_SHIPMENT_ID_VALUE</td>
+	<td>//span[@title="Identification Number"]//..//strong[contains(@class,'header-info-value') and normalize-space(text()='${key_id}')]</td>
+	 <td></td>
+</tr>
+<tr>
+	<td>ORDER_AND_SHIPMENT_ERC_VALUE</td>
+	<td>//span[@title="External Reference Code"]//..//strong[contains(@class,'header-info-value') and normalize-space(text()='${key_erc}')]</td>
+	 <td></td>
+</tr>
```
[the remark this answers is not in the document]
Comment (baharturk) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceOrders.path:107]: As agreed in a call, this remains as it is.

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:commerce-functional

**Comment by baharturk** [comment]
ci:forward

**Pull request closed without merging**